### PR TITLE
Update UG Features before Add

### PR DESCRIPTION
**Pull request by xCOLOURx**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -105,70 +105,75 @@ As ModCraft writes to files within the directory it is in, we recommend placing
...
 * All commands are **case-sensitive**.
 
-* Words in `UPPER_CASE` are **compulsory parameters** to be supplied by the user.<br>
-  e.g. in `delete MODULE`, `MODULE` is a parameter which can be used as `delete CFG1002`.
+* Words in `UPPER_CASE` are **compulsory parameters** supplied by you.<br>
+  e.g. in `delete MODULE`, `MODULE` is a parameter which can be used as `delete CS1231S`.
 
-* Parameters in square brackets denote **optional parameters**.<br>
-  e.g. `edit [y/YEAR]` means that specifying `y/YEAR` is optional. However, if `y/` is keyed in by the user, `YEAR` has to be specified.
+* Parameters in square brackets denotes **optional parameters**.<br>
```
Comment (rocketninja7) [inline docs/UserGuide.md:115]: denote was correct

File: docs/UserGuide.md [hunk 2]
```diff
@@ -105,70 +105,75 @@ As ModCraft writes to files within the directory it is in, we recommend placing
...
+  e.g. `edit [y/YEAR]` means that specifying `y/YEAR` is optional. However, if you keyed `y/`, `YEAR` has to be specified.
```
Comment (rocketninja7) [inline docs/UserGuide.md:116]: keyed in

File: docs/UserGuide.md [hunk 3]
```diff
@@ -105,70 +105,75 @@ As ModCraft writes to files within the directory it is in, we recommend placing
...
 
-* Grades and semesters as parameters are **case-sensitive**. Only grades and semesters in **capital letters** are allowed.<br>
+* Grade and Semester are **case-sensitive parameters**. Only grades and semesters in **capital letters** are allowed.<br>
```
Comment (rocketninja7) [inline docs/UserGuide.md:118]: Maybe use code block and all caps

File: docs/UserGuide.md (hunk 1 above)
Comment (xCOLOURx) [reply docs/UserGuide.md:115]: fixed

File: docs/UserGuide.md (hunk 2 above)
Comment (xCOLOURx) [reply docs/UserGuide.md:116]: fixed

**Pull request merged**